extensions/map-layers-formats/package.json

```diff
@@ -7,14 +7,16 @@
   "typings": "lib/cjs/map-layers-formats",
   "license": "MIT",
   "scripts": {
-    "build": "npm run -s build:cjs && npm run -s build:esm",
+    "build": "npm run -s copy:locale && npm run -s build:cjs && npm run -s build:esm",
```

Comment:
add an npmignore file to make sure you deliver what you are exepecting to

Reply:
done